debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -534,6 +493,10 @@ public String buildSelectWithRowLimits(TableId tableId, int limit, String projec
 
     @Override
     public String quotedTableIdString(TableId tableId) {
-        return "[" + tableId.schema() + "].[" + tableId.table() + "]";
+        return "[" + tableId.catalog() + "].[" + tableId.schema() + "].[" + tableId.table() + "]";
```

Comment:
This makes me wonder: will there be a change to the type schemas in emitted message names in "multi-DB" mode? I suppose the answer is yes, schema names will be prefixed with the DB name?

Reply:
Yes, it was implemented as part of https://github.com/sugarcrm/debezium/pull/30.

Reply:
Ok, gotcha. I suppose a similar change will happen to topic names which also need to include the DB name.